packages/fx-core/templates/plugins/resource/simpleauth/bicep/module.template.bicep

```diff
@@ -0,0 +1,16 @@
+
+module simpleAuthProvision '{{fx-resource-simple-auth.modules.simpleAuthProvision.path}}' = {
+  name: 'simpleAuthProvision'
+  params: {
+    simpleAuthServerFarmsName: simpleAuth_serverFarmsName
+    simpleAuthWebAppName: simpleAuth_webAppName
+    sku: simpleAuth_sku
+    AADClientId: AADClientId
+    AADClientSecret: AADClientSecret
+    applicationIdUri: applicationIdUri
+    {{#contains 'fx-resource-frontend-hosting' plugins}}
+    frontendHostingStorageEndpoint: {{../fx-resource-frontend-hosting.outputs.endpoint}}
```

Comment:
does simpleAuth needs aware frontend plugin?

Reply:
Yes, this is the contract between the two resource plugin (simple auth / frontend hosting).
Simple auth resource plugin does not know the actual value of `fx-resource-frontend-hosting.outputs.endpoint`, it just know that if frontend-hosting resource plugin exists, it should have some changes on its bicep files.
While solution plugin will help to replace the placeholder with the real value.